packages/gql/codegen.ts

```diff
@@ -82,6 +83,34 @@ export const SUBGRAPHS = {
       return this.endpoints()[network];
     },
   },
+  [Subgraph.ChainsBlocks]: {
+    name: Subgraph.ChainsBlocks,
+    endpoints() {
+      return {
+        [Network.Ethereum]:
+          "https://api.thegraph.com/subgraphs/name/blocklytics/ethereum-blocks",
+        //TODO: BAL-659 substitute Sepolia, Gnosis and Optimism, not functional yet
```

Comment:
The task of creating it seems to have been finished ([BAL-131](https://linear.app/bleu-llc/issue/BAL-131/deploy-contracts-in-all-networks-that-balancer-is-deployed)), why are they different?

Reply:
@Rawallon the task you mentioned is for metadata custom subgraph